src/coverlet.console/Program.cs

```diff
@@ -57,6 +57,12 @@ static int Main(string[] args)
                     logger.Level = verbosity.ParsedValue;
                 }
 
+                // We add default exclusion filter if no specified
+                if (excludeFilters.Values.Count == 0)
```

Comment:
Just curious, why did we chose to include if `Count == 0` instead of `excludeFilters.Values.Contains("[xunit*]*"`?
Since i cannot think an scenario in which we want to include this dlls, and maybe we forgot to include it in our custom `exclude` filters. Or we missincluded it using the wrong wildcards or whatever
@MarcoRossignoli

Reply:
Comment says "We add default exclusion filter if no specified" but you are not adding the `CoverletConstants. DefaultExcludeFilter` here, but the `xunit` one, which i find confusing

Reply:
> Since i cannot think an scenario in which we want to include this dlls

The idea is to remove only in default config...after first include users win.

Reply:
When are you adding the `CoverletConstants. DefaultExcludeFilter` btw?
Not sure about the code, but if this later goes into `CoverletSettingsParser` then it will never reach the code you added in that file (the if statement), since you add this before reaching that part, and then all it does is add the constants

Reply:
`CoverletConstants.DefaultExcludeFilter` it's needed only by collector.

Reply:
Today we've 3 entry point/driver for coverlet console/msbuild/collector they behave in a different way on defaults.

Reply:
check manual tests FYI
https://github.com/tonerdo/coverlet/pull/462#issuecomment-500449165
https://github.com/tonerdo/coverlet/pull/462#issuecomment-500450742